packages/@okta/i18n/src/properties/login.properties

```diff
@@ -533,7 +533,7 @@ primaryauth.submit = Sign In
 primaryauth.newUser.tooltip=This is the first time you are connecting to {0} from this browser
 primaryauth.newUser.tooltip.close=Close
 oktaVerify.description = To access this resource, your organization requires you to sign in using your device.
-oktaVerify.button = Sign in using Okta Verify on this device
+oktaVerify.button = Sign in with Okta FastPass
```

Comment:
Nit I guess it is better to rename the key to oie.oktaVerify.fastpass.button so developers dont confuse with oktaVerify in v1 or elsewhere.